serverManager/common/source/SessionServerAppManager.cpp

```diff
@@ -46,26 +46,32 @@ SessionServerAppManager::~SessionServerAppManager()
     m_eventThread.reset();
 }
 
+bool SessionServerAppManager::initiateApplication(const std::string &appId,
+                                                  const firebolt::rialto::common::SessionServerState &state,
+                                                  const firebolt::rialto::common::AppConfig &appConfig)
+{
+    RIALTO_SERVER_MANAGER_LOG_INFO("RialtoServerManager requests to launch %s with initial state: %s", appId.c_str(),
+                                   toString(state));
+    if (state != firebolt::rialto::common::SessionServerState::NOT_RUNNING && !isSessionServerLaunched(appId))
+    {
+        return addSessionServer(appId, state, appConfig);
+    }
+    RIALTO_SERVER_MANAGER_LOG_ERROR("Initialization of %s failed.", appId.c_str());
```

Comment:
minor: give some indication of the reason? i.e. bad state or server already running

Reply:
changed.